CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Add [google-map-clustering-example](https://github.com/istarkov/google-map-clustering-example)
 
+###[Unreleased]
```

Comment:
@jonathanweiss Can we move this section to the bottom? And also, no need of add `Unreleased` as I am going to release it very soon after merged.